corehq/apps/translations/views.py

```diff
@@ -21,10 +23,27 @@
 from corehq.apps.domain.views import BaseDomainView
 from custom.icds.translations.integrations.exceptions import ResourceMissing
 from custom.icds.translations.integrations.transifex import Transifex
-from custom.icds.translations.integrations.utils import get_file_content_from_workbook
+from custom.icds.translations.integrations.utils import get_file_content_from_workbook, \
+    transifex_details_available_for_domain
 
 
-class ConvertTranslations(BaseDomainView):
+class BaseTranslationsView(BaseDomainView):
+    @property
+    def page_context(self):
+        transifex_details_available = transifex_details_available_for_domain(self.domain)
```

Comment:
This might be nice as a memoized property, since it's referenced in a few other places